### PR TITLE
Enhance landing readability and change style

### PR DESCRIPTION
**Pull request by Sigevio**

### CODE REVIEW

**Review by pavelskipenes — COMMENTED** [review]
So, I'm no front end developer so I'll not comment on the code part too much. 

A new picture looks great. Better on desktop than mobile but still pretty good.
Font on heading looks generic to me. Maybe try something else? I don't know much about fonts but Roboto and Cantarell are pretty nice :smile:

**Comment by pavelskipenes** [comment]
`old-hero.jpg` can probably be removed from this PR

**Comment by Sigevio** [comment]
Agreed, mobile is working, but looks kind of bad.
Every font in this project is Roboto. Can try to mix it up with something else on headers.

File: i18n/translations/fr.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "store": "Boutique"
   },
   "home": {
-    "header": "Bienvenue à NTNUI Swimming!",
-    "headerDescription": "- Nous rendons la natation norvégienne plus amusante!",
+    "header": "Bienvenue à NTNUI Swimming",
+    "headerDescription": "- Nous rendons la natation norvégienne plus amusante",
```
Comment (pavelskipenes) [inline i18n/translations/fr.json:14]: what's this french shit doing here :rofl:

Comment (Sigevio) [reply i18n/translations/fr.json]: Foreshadowing;)

File: i18n/translations/no.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "store": "Nettbutikk"
   },
   "home": {
-    "header": "Velkommen til NTNUI Svømming",
+    "header": "Svømmegruppa",
```
Comment (pavelskipenes) [inline i18n/translations/no.json:38]: We should really not be using "svømmegruppa". It's not our official name, it's more dialect than official language and it's harder for internationals to understand "svømmegruppa" compared to "ntnui svømming".

Comment (Sigevio) [reply i18n/translations/no.json:38]: How is "Swimming group" any better?

Comment (pavelskipenes) [reply i18n/translations/no.json:38]: true

Comment (pavelskipenes) [reply i18n/translations/no.json:38]: Alright, as long as we stick to ntnui swimming in official documents I think we can allow ourselves to use alternate names. At least until someone complains :crossed_fingers:

Comment (Sigevio) [reply i18n/translations/no.json:38]: Sounds good XD

**Review by pavelskipenes — APPROVED** [review]

**Pull request merged**